{{ cookiecutter.__package_name_kebab_case }}/README.md

```diff
@@ -22,53 +22,60 @@ To serve this app, run `docker compose up app` and open [localhost:8000](http://
 {% if cookiecutter.continuous_integration == "GitLab" -%}
 1. [Generate an SSH key](https://docs.gitlab.com/ee/ssh/README.html#generate-an-ssh-key-pair) and [add the SSH key to your GitLab account](https://docs.gitlab.com/ee/ssh/README.html#add-an-ssh-key-to-your-gitlab-account).
 1. Configure SSH to automatically load your SSH keys:
-    ```sh
-    cat << EOF >> ~/.ssh/config
-    Host *
-      AddKeysToAgent yes
-      IgnoreUnknown UseKeychain
-      UseKeychain yes
-    EOF
-    ```
+
+        ```sh
+        cat << EOF >> ~/.ssh/config
+        Host *
+          AddKeysToAgent yes
+          IgnoreUnknown UseKeychain
+          UseKeychain yes
+        EOF
+        ```
```

Comment:
No need to change the indent to 8 spaces: 4 spaces is enough, see e.g. [1].

[1] https://gist.github.com/clintel/1155906